phpmd.xml

```diff
@@ -1,24 +1,22 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<ruleset name="PHPMD rule set"
+<?xml version="1.0"?>
+<ruleset name="My first PHPMD rule set"
          xmlns="http://pmd.sf.net/ruleset/1.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
-    http://pmd.sf.net/ruleset_xml_schema.xsd"
+                       http://pmd.sf.net/ruleset_xml_schema.xsd"
          xsi:noNamespaceSchemaLocation="
-    http://pmd.sf.net/ruleset_xml_schema.xsd"
->
-    <rule ref="rulesets/cleancode.xml" >
-        <exclude name="ElseExpression" />
-        <exclude name="StaticAccess" />
-        <exclude name="BooleanArgumentFlag" />
-    </rule>
+                       http://pmd.sf.net/ruleset_xml_schema.xsd">
+    <description>
+        My custom rule for PR tracker
```

Comment:
... with a real description

Reply:
real description for all rules and exclude